lib/developer_portal/lib/liquid/tags/cdn_asset.rb

```diff
@@ -13,45 +13,51 @@ class CdnAsset < Liquid::Tags::Base
 
       def initialize(tag_name, file, tokens)
         super
-        @file = File.join(cdn_host, file).strip # strip because we are allowing the path not enclosed between quotes
+        @file = File.join('/dev-portal-assets', file).strip # strip because we are allowing the path not enclosed between quotes
       end
 
-      desc %{
+      desc %(
         Provides the desired asset file.
-   
+
         The assets you can use here are limited to:
-        
+
         Font Awesome:
         - font-awesome/4.3.0/css/font-awesome.css
         - font-awesome/4.3.0/css/font-awesome.min.css
 
-        jQuery 1.11.4:
-        - jquery-ui/1.11.4/jquery-ui.css 
-        - jquery-ui/1.11.4/jquery-ui.min.js
-
         jQuery 1.7.1:
         - jquery/1.7.1/jquery.min.js
-      }
+
+        jQuery 3.5.0:
+        - jquery/3.5.0/jquery.min.js
+
+        jQuery UI 1.11.4:
+        - jquery-ui/1.11.4/jquery-ui.css
+        - jquery-ui/1.11.4/jquery-ui.min.js
+
+        SwaggerUI 2.0.23
+        - swagger-ui/2.0.23/swagger-ui.js
+
+        SwaggerUI 2.1.3
+        - swagger-ui/2.1.3/swagger-ui.js
+
+        SwaggerUI 2.2.10
+        - swagger-ui/2.2.10/swagger-ui.js
+      )
 
       def render(context)
         action_view = context.registers[:controller].view_context
+        file_url = File.join(context.registers[:controller].helpers.rails_asset_host_url, @file)
```

Comment:
Maybe extract `context.registers[:controller]` to a variable to make CodeClimate happy?

Reply:
Done!